src/box/rt_lua_call_access.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <zlib.h>
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+#define is_super(u) ((u)->roles.m[0] == 20u)
+#define is_universe(s, l) ((l) == 0 || (s)[0] == '\0')
+#define RT_ACCESS 1
+#define RT_NO_ACCESS 0
```

Comment:
AFAIS, we have two general approaches in tarantool:

1. Use `bool` return value.
2. Use `int`, where 0 is success and -1 (with setting a diagnostics area) is an error.

Since we don't set a diag here, I would use booleans.

Reply:
I decided not to use `RT_ACCESS` and others, but to use `bool` directly in `rt_access_check`.